config/initializers/logjam_agent.rb

```diff
@@ -71,4 +71,13 @@ module LogjamAgent
   # all exceptions below a log level of Logger::ERROR. Logjam itself can then
   # display those soft exceptions differently. Defaults to `true`.
   # self.split_hard_and_soft_exceptions = true
+
+  # Patch the STDOUT Forwarder to send to Rails' logger instead, since stdout
+  # seems to get lost somewhere.
+  class STDOUTForwarder
+    def forward(data, options={})
+      msg = LogjamAgent.json_encode_payload(data)
+      Rails.logger.info msg
```

Comment:
Oh god. We should keep an eye on the production log after this is live and make sure we're not pelting it with noise.